src/backend/distributed/deparser/deparse_database_stmts.c

```diff
@@ -136,44 +143,52 @@ AppendGrantOnDatabaseStmt(StringInfo buf, GrantStmt *stmt)
 }
 
 
+static bool
+AppendBasicAlterDatabaseOptions(StringInfo buf, DefElem *def, bool
+								prefix_appended_for_basic_options, char *dbname)
+{
+	if (!prefix_appended_for_basic_options)
```

Comment:
we don't use snake_case, please see https://github.com/citusdata/citus/pull/7347